app/changelog/kube-state-metrics.yaml

```diff
@@ -1,3 +1,9 @@
+- version: 1.0.5
+  componentVersion: 1.9.5
+  description: Updated to upstream kube-state-metrics v1.9.5.
+  kind: changed
+  urls:
+    - https://github.com/giantswarm/kube-state-metrics-app/blob/master/CHANGELOG.md#v105
```

Comment:
```suggestion
```

As agreed via [Slack](https://gigantic.slack.com/archives/C02521EC0/p1585238800011300), extracted this to separate PR https://github.com/giantswarm/releases/pull/195 so it can be used in other releases crafted in parallel with 9.0.1.